src/Analyzers/VisualBasic/Tests/RemoveUnnecessaryCast/RemoveUnnecessaryCastTests.vb

```diff
@@ -31,14 +31,46 @@ End Module
             Await TestMissingAsync(markup)
         End Function
 
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessaryCast)>
+        Public Async Function TestDontRemoveCastSimpleArgument1() As Task
+            Dim markup =
+<File>
+Option Strict On
+Imports System.Drawing
+Module M
+    Sub Main()
+        Dim x As New Point([|CInt(System.Math.Floor(1.1))|], 1)
+    End Sub
+End Module
+</File>
+
+            Await TestMissingAsync(markup)
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnnecessaryCast)>
+        Public Async Function TestDontRemoveCastSimpleArgument2() As Task
+            Dim markup =
+<File>
+Option Strict On
+Imports System.Drawing
+Module M
+    Sub Main()
+       Dim y As New Point([|CInt(1.1), 1)|]
```

Comment:
the selection span is a bit strange here.  was it intentional?

Reply:
No it's just lack of understand how to select a span. I will fix.